README.md

```diff
@@ -55,28 +55,35 @@ UI, ready for you to install.
 The graphical SDK will allow you to quickly build and test simple databox apps. The current version of SDK run by default in `dev` mode. To start the sdk run:
 ```
 ./databox-start sdk
-```        
+```
 The SDK web UI is available at http://127.0.0.1:8086
 
-To stop the SDK run: 
+To stop the SDK run:
 ```
 ./databox-stop sdk
 ```
 
-Databox has a number of platform components, divided into two parts:  Core and User components.
+When you start in development mode only the `core-components` are built from source. If you wish to develop one of the available apps or drivers then you can add them to you local install using:
+
+```
+./databox-install-component driver-os-monitor
+```
+
+This will download and build the code on your machine and upload the Databox manifest to your local app store. You can also use this with your repositories  and forks using:
+
+```
+./databox-install-component [GITHUB_USERNAME]/[GITHUB_REPONAME]
+```
 
 ### Core Components
 
-* [Databox container manager](https://github.com/me-box/core-container-manager)
+Databox has a number of platform components, divided into two parts:  Core and User components.
 
+* [Databox container manager](https://github.com/me-box/core-container-manager)
 * [databox-arbiter](https://github.com/me-box/core-arbiter)
-
 * [databox-export-service](https://github.com/me-box/core-export-service)
-
 * [databox-store-json](https://github.com/me-box/store-json)
-
 * [databox-store-timeseries](https://github.com/me-box/store-timeseries)
-
 * [databox-app-server](https://github.com/me-box/platform-app-server) Server for storing and serving databox manifests
 
```

Comment:
Worth adding short descriptions (and consistent naming format) to all of these

Reply:
Should this be a separate issue for documentation? Rather than part of this PR

Reply:
I'll create a new PR for this.

Reply:
Cheers 👍